src/brain/issueLabelHandler/followups.ts

```diff
@@ -45,7 +45,6 @@ export async function updateCommunityFollowups({
   const reasonsToDoNothing = [
     isNotInARepoWeCareAboutForFollowups,
     isNotFromAnExternalOrGTMUser,
-    isNotWaitingForCommunity,
```

Comment:
There's no reason to limit this to issues that have `Waiting for: Community` label. Any issue that has an external member comment on it can be added to `Issues Someone Else Cares About` project and have the `Waiting for: Product Owner` label applied